### PR TITLE
PoC : regex/s are vulnerable for validation

### PR DESCRIPTION
**Pull request by Lukmanern**
TL;DR : request to remove some regex-base validation, like email regex validation.
Background : So many CVE's cause by Regex (See : https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=regex )
1. Please use Regex wisely, there is email validation using regex and Go has std package/s like validation/v10 or net/mail .
2. Use Regex for validating data with limited character types. Example to validate valid names in id_ID where characters are limited to A-Za-z, with the addition of space, dash and/or a single tick/quote, you can use a regex pattern for validate that name-type.
3. If there isn't a regex pattern available or too hard to create for validating specific data, consider using string replacement and validating each separated part of the data/value, it's more safe and clear to see.
4. And please provide datasets as well for testing each regex base validation/s .

note : user@example.com. (with dot in the end) is valid email for regex validation.

### CODE REVIEW

**Review by josestg — COMMENTED** [review]
Can you resolve the issue as well, @Lukmanern ? In my opinion, you can put these fuzzy tests into a test file, mas.

**Comment by Lukmanern** [comment]
I can, I will resolve ASAP, in 4/5 days.
